### PR TITLE
Add remaining ProSiebenSat.1 brands

### PR DESCRIPTION
**Pull request by vikramadityasinghs**
Issue #1255 
Added json files for the remaining 8 websites, brands and apps run by Sat.1 in /companies.

### CODE REVIEW

**Review by vikramadityasinghs — COMMENTED** [review]
hi @mal-tee, I have made all the required changes.
if there wasn't a verified source of a fax number I omited it from the json file.

**Review by vikramadityasinghs — COMMENTED** [review]
updated sixx.json

File: companies/kabeleins.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "entertainment"
+    ],
+    "name": "kabeleins.de",
+    "address": "Seven.One Entertainment Group GmbH\nMedienallee 7\nD-85774 Unterföhring\nGermany",
+    "phone": "+49 89 950710",
+    "fax": "+49 89 3977199",
```
[the remark this answers is not in the document]
Comment (mal-tee) [reply companies/kabeleins.json]: Sorry for taking so long -- yes, if there is no source we should omit it.

File: companies/joyn-gmbh.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "slug": "joyn-gmbh",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "entertainment"
+    ],
+    "name": "Joyn GmbH",
+    "address": "Ridlerstrasse 57\n80339 Munich\nGermany",
```
Comment (mal-tee) [inline companies/joyn-gmbh.json:10]:
```suggestion
    "address": "Ridlerstrasse 57\n80339 München\nGermany",
```

File: companies/joyn-gmbh.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "phone": "+49 800 4395696",
+    "fax": "+49 89 3977199",
```
Comment (mal-tee) [inline companies/joyn-gmbh.json:12]:
```suggestion
    "fax": "+49 89 943977199,
```
that's the number from the "impressum"

File: companies/kabeleins.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    ],
+    "name": "Seven.One Entertainment Group GmbH",
+    "address": "Medienallee 7\n85774 Unterföhring\nGermany",
+    "phone": "+49 89 950710",
+    "email": "datenschutz@kabeleins.de",
+    "web": "https://www.kabeleins.de/",
+    "sources": [
+        "https://www.kabeleins.de/service/nutzungsbedingungen/datenschutz",
+        "https://www.kabeleins.de/service/impressum",
+        "https://www.sevenonemedia.de/impressum"
```
Comment (mal-tee) [inline companies/kabeleins.json:17]:
```suggestion
```

File: companies/kabeleinsdoku.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
+{
+    "slug": "kabeleinsdoku",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "entertainment"
+    ],
+    "name": "Seven.One Entertainment Group GmbH",
```
Comment (mal-tee) [inline companies/kabeleinsdoku.json:9]: please add a `runs`  with "Kabel Eins Doku", to improve the value of this record

File: companies/prosieben.json [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+        "entertainment"
+    ],
+    "name": "Seven.One Entertainment Group GmbH",
+    "address": "Medienallee 7\nD-85774 Unterföhring\nGermany",
+    "phone": "+49 89 950710",
+    "email": "datenschutz@prosieben.de",
+    "web": "https://www.prosieben.de/",
+    "sources": [
+        "https://www.prosieben.de/service/impressum",
+        "https://www.sevenonemedia.de/impressum",
```
Comment (mal-tee) [inline companies/prosieben.json:16]:
```suggestion
```

File: companies/prosiebenmaxx.json [hunk 7]
```diff
@@ -0,0 +1,20 @@
+{
+    "slug": "prosiebenmaxx",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "entertainment"
+    ],
+    "name": "ProSiebenSat.1 Digital GmbH",
+    "address": "Medienallee 4\nD-85774 Unterföhring\nGermany",
```
Comment (mal-tee) [inline companies/prosiebenmaxx.json:10]:
```suggestion
    "address": "Medienallee 4\n85774 Unterföhring\nGermany",
```

File: companies/prosiebenmaxx.json [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+    "phone": "+49 89 950710",
+    "email": "datenschutz@prosiebenmaxx.de",
+    "web": "https://www.prosiebenmaxx.de/",
+    "sources": [
+        "https://www.prosiebenmaxx.de/service/datenschutzbestimmungen",
+        "https://www.prosiebenmaxx.de/service/impressum"
```
Comment (mal-tee) [inline companies/prosiebenmaxx.json:16]:
```suggestion
```

File: companies/ran.json [hunk 9]
```diff
@@ -0,0 +1,21 @@
+{
+    "slug": "ran",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "entertainment"
+    ],
+    "name": "Seven.One Sports GmbH",
+    "address": "Medienallee 7\n85774 Unterföhring\nGermany",
```
Comment (mal-tee) [inline companies/ran.json:10]:
```suggestion
    "address": "c/o ProSiebenSat.1 Media SE\nMedienallee 7\n85774 Unterföhring\nGermany",
```

File: companies/sat1gold.json [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+{
+    "slug": "sat1gold",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "entertainment"
+    ],
+    "name": "Seven.One Entertainment Group GmbH",
+    "address": "Medienallee 7\nD-85774 Unterföhring\nGermany",
```
Comment (mal-tee) [inline companies/sat1gold.json:11]:
```suggestion
    "address": "Medienallee 7\n85774 Unterföhring\nGermany",
```

File: companies/sat1gold.json [hunk 11]
```diff
@@ -0,0 +1,22 @@
...
+    "phone": "+49 89 950710",
+    "email": "datenschutz@sat1gold.de",
+    "web": "https://www.sat1gold.de/",
+    "sources": [
+        "https://www.sat1gold.de/service/datenschutz-cookies/datenschutzbestimmungen",
+        "https://www.sevenonemedia.de/impressum",
+        "https://www.sat1gold.de/service/impressum"
```
Comment (mal-tee) [inline companies/sat1gold.json:18]:
```suggestion
```

File: companies/sixx.json [hunk 12]
```diff
@@ -0,0 +1,21 @@
...
+    "slug": "sixx",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "entertainment"
+    ],
+    "name": "Fem Media GmbH",
+    "address": "Gutenbergstr. 3\n85774 Unterföhring\nGermany",
+    "phone": "+49 89 95072898",
```
Comment (mal-tee) [inline companies/sixx.json:11]:
```suggestion
    "phone": "+49 89 950710",
```

**Review by mal-tee — CHANGES_REQUESTED** [review]
Thank you! I've left some comments.

**Comment by mal-tee** [comment]
I took the liberty and merged master, which includes better tests :)

**Review by vikramadityasinghs — COMMENTED** [review]
I've made the particular changes you requested, sorry I will keep it in mind 👍 @mal-tee

**Review by vikramadityasinghs — COMMENTED** [review]
Fixing some minute errors

**Review by vikramadityasinghs — COMMENTED** [review]
fixed the errors

File: companies/kabeleinsdoku.json [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+    "runs": [
+        "www.kabeleinsdoku.de",
+        "www.kabeleinsdoku.at",
+        "www.kabeleinsdoku.ch"
```
Comment (mal-tee) [inline companies/kabeleinsdoku.json:13]:
```suggestion
        "Kabel Eins Doku"
```

**Review by mal-tee — CHANGES_REQUESTED** [review]

**Review by mal-tee — APPROVED** [review]
Thank you! I'll wait for the tests and merge.

**Comment by vikramadityasinghs** [comment]
thanks a lot! @mal-tee hope to work  on this repo again :smile:

**Pull request merged**